.github/workflows/imagebuild.yml

```diff
@@ -0,0 +1,140 @@
+name: Build and Push Georust Docker Images
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+      - trying
+  pull_request:
+  merge_group:
+
+env:
+  LIBPROJ_VERSION: 9.4.0
+  BASE_IMAGE_NAME: libproj-builder
+
+jobs:
+  build_base_image:
+    name: Build the base image for geo and proj
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        rust_version: [1.77]
+
+    steps:
+    - name: Check out repository
+      uses: actions/checkout@v4
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v3
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v3
+
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Set Up Builder
+      run: |
+        docker buildx create --name mybuilder
+        docker buildx use mybuilder
+        docker buildx inspect --bootstrap
+
+    - name: Build and export base image to Docker
+      uses: docker/build-push-action@v5
+      with:
+        file: ./dockerfiles/${{ env.BASE_IMAGE_NAME }}
+        push: false
+        load: true
+        platforms: linux/amd64
+        tags: ghcr.io/${{ github.repository_owner }}/${{ env.BASE_IMAGE_NAME }}:proj-${{ env.LIBPROJ_VERSION }}-rust-${{ matrix.rust_version }}
```

Comment:
Ah - I see the tag scheme here. Looks like it's the same as before.

Reply:
Yep, it's the same!